src/script/components/calling/VideoControls/VideoControls.tsx

```diff
@@ -0,0 +1,791 @@
+  const [isCallViewOpen, setIsCallViewOpen] = useState(false);
+
+  const showToggleVideo =
+    isVideoCallingEnabled &&
+    (call.initialType === CALL_TYPE.VIDEO || conversation.supportsVideoCall(call.isConference));
+
+  const showSwitchMicrophone = audioinput.length > 1;
+
+  const callViewOptions = [
+    {
+      label: t('videoCallOverlayViewModeLabel'),
+      options: [
+        {
+          label: t('videoCallOverlayViewModeAll'),
+          value: CallViewTab.ALL,
+        },
+        {
+          label: t('videoCallOverlayViewModeSpeakers'),
+          value: CallViewTab.SPEAKERS,
+        },
+      ],
+    },
+  ];
+
+  const selectedCallViewOption =
+    callViewOptions[0].options.find(option => option.value === activeCallViewTab) ?? callViewOptions[0].options[0];
+
+  const audioOptions = [
+    {
+      label: t('videoCallaudioInputMicrophone'),
+      options: audioinput.map((device: MediaDeviceInfo | ElectronDesktopCapturerSource) => {
+        return isMediaDevice(device)
+          ? {
+              label: device.label,
+              value: `${device.deviceId}-input`,
+              dataUieName: `${device.deviceId}-input`,
+              id: device.deviceId,
+            }
+          : {
+              label: device.name,
+              value: `${device.id}-input`,
+              dataUieName: `${device.id}-input`,
+              id: device.id,
+            };
+      }),
+    },
+    {
+      label: t('videoCallaudioOutputSpeaker'),
+      options: audiooutput.map((device: MediaDeviceInfo | ElectronDesktopCapturerSource) => {
+        return isMediaDevice(device)
+          ? {
+              label: device.label,
+              value: `${device.deviceId}-output`,
+              dataUieName: `${device.deviceId}-output`,
+              id: device.deviceId,
+            }
+          : {
+              label: device.name,
+              value: `${device.id}-output`,
+              dataUieName: `${device.id}-output`,
+              id: device.id,
+            };
+      }),
+    },
+  ];
+
+  const [selectedAudioOptions, setSelectedAudioOptions] = useState(() =>
+    [currentMicrophoneDevice, currentSpeakerDevice].flatMap(
+      (device, index) => audioOptions[index].options.find(item => item.id === device) ?? audioOptions[index].options[0],
+    ),
+  );
+
+  const updateAudioOptions = (selectedOption: string, input: boolean) => {
+    const microphone = input
+      ? audioOptions[0].options.find(item => item.value === selectedOption) ?? selectedAudioOptions[0]
+      : selectedAudioOptions[0];
+    const speaker = !input
+      ? audioOptions[1].options.find(item => item.value === selectedOption) ?? selectedAudioOptions[1]
+      : selectedAudioOptions[1];
+
+    setSelectedAudioOptions([microphone, speaker]);
+    switchMicrophoneInput(microphone.id);
+    switchSpeakerOutput(speaker.id);
+  };
+
+  const isBlurredBackgroundEnabled = Config.getConfig().FEATURE.ENABLE_BLUR_BACKGROUND;
```

Comment:
You can assign rest of the used features here, so code wil be more clean :)

```suggestion
  const {ENABLE_BLUR_BACKGROUND: isBlurredBackgroundEnabled} = Config.getConfig().FEATURE;
```

Reply:
this was the existing code from FullscreenVideoCall :) updated